src/components/CreateVmWizard/steps/Networking.js

```diff
@@ -132,14 +142,26 @@ class Networking extends React.Component {
         },
         editView: (value, { rowData }) => {
           const row = this.state.editing[rowData.id]
+          const nicNameErrFlag = this.state.editedRowNameErrorsFlag
```

Comment:
This is inconsistent with how the edit data is being stored.  The state data is setup to potentially allow simultaneous editing of rows.  This probably won't ever be the case, so the error flags can be done without regards to supporting multiple edit rows being active at the same time.

Something like `this.state.editingErrors.nicInvalidName` and/or `this.state.editingErros.nicDuplicateName` would be easier to understand at first reading.